### ISSUE TITLE
Build error back_inserter identifier not found in utf8.cpp 

### ISSUE THREAD

**Issue by eskin** [arvidn/libtorrent#205]
Version 1.0.6
Build fail on MSVC 2010

````
src\utf8.cpp(61) : error C2039: back_inserter: не является членом "std"
src\utf8.cpp(61) : error C3861: back_inserter: идентификатор не найден
src\utf8.cpp(78) : error C2039: back_inserter: не является членом "std"
src\utf8.cpp(78) : error C3861: back_inserter: идентификатор не найден
````

Solution:
Add this line to utf8.cpp:
````
#include <iterator>
````

**Comment by aldenml** [comment]
That's fixed with this commit: https://github.com/arvidn/libtorrent/commit/f01a9bd8d056d47925056a5c32458be12cc0f570

**Comment by arvidn** [comment]
and will be released as part of 1.0.7 any day now
